jpegtran/lib.py

```diff
@@ -260,4 +260,17 @@ def crop(self, x, y, width, height):
         options.r.x = x
         options.r.y = y
         options.options = lib.TJXOPT_CROP
-        return options
+        return options, 0
+
+    @jpegtran_op
+    def progressive(self):
+        options = self._get_transformoptions()
+        options.options = lib.TJXOPT_PROGRESSIVE
+        options.options |= lib.TJXOPT_COPYNONE
+        return options, 16384
```

Comment:
I'm not sure what this `16384` means.  Maybe it could be documented in a docstring?   Note: its forcing `flag` to be added elsewhere.

Reply:
https://github.com/libjpeg-turbo/libjpeg-turbo/blob/ae87a958613b69628b92088b313ded0d4f59a716/turbojpeg.h#L420

#define TJFLAG_PROGRESSIVE  16384